src/Infrastructure/Workflow/ProjectWorkflow.php

```diff
@@ -5,7 +5,7 @@
  * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
  */
 
-namespace SprykerSdk\Sdk\Core\Application\Service;
+namespace SprykerSdk\Sdk\Infrastructure\Workflow;
```

Comment:
Should be moved to the application layer along with the `WorkflowTransitionResolverRegistry`